Capstone/Friends/FriendDetailsViewController.m

```diff
@@ -0,0 +1,128 @@
+//
+//  FriendDetailsViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 8/1/22.
+//
+#import <Parse/Parse.h>
+#import "ParseCollege.h"
+#import "Parse/PFImageView.h"
+#import "FriendDetailsViewController.h"
+#import "FriendMatchesCell.h"
+#import "DetailsViewController.h"
+ 
+@interface FriendDetailsViewController ()<UITableViewDelegate, UITableViewDataSource>
+@property (weak, nonatomic) IBOutlet UILabel *username;
+@property (weak, nonatomic) IBOutlet UIButton *sendFriendRequest;
+@property (weak, nonatomic) IBOutlet UIButton *cancelFriendRequest;
+@property (weak, nonatomic) IBOutlet UIImageView *userImage;
+@property (weak, nonatomic) IBOutlet UIButton *friends;
+@property (weak, nonatomic) IBOutlet UITableView *tableView;
+@end
+ 
+@implementation FriendDetailsViewController {
+    NSArray *usersMatchedColleges;
+    NSMutableArray *friends;
+    NSMutableArray *localFriends;
+    PFRelation *matchesRelation;
+    PFUser *currentUser;
+}
+ 
+- (void)viewDidLoad {
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    self.username.text = self.user.username;
+    self->localFriends = [[NSMutableArray alloc] init];
```

Comment:
we don't need this init since we're reassigning this method in L37!

Reply:
makes sense. resolved